helpers/utils.h

```diff
@@ -25,7 +25,9 @@
 #include <syscall.h>
 
```

Comment:
The architectural setup of Thunder/ThunderNanoservices is to have layers. Higher layers should only interact with lower layers. components in the same layer should not interact with the next component. The whole helper pardigm used is contradicting this architectural concept. 
TheSideCar approach has more plugins which are closed source. With this setup, those modules (if they wanted to use this helper stuff had to create a dependency on this reporsitory. This is what we would like to avoid hence why you have "supporting" libraries in thunder, (like e.g. Tracing for Tracing), Plugins for Plugins, WebSockets for http comunnication etc.
Functionality in this directory, if usefull should end up in the Thunder repository in the respective modules (Core/CryptAlgo/Tracing/WebSocket/com etc) to maintain a clean "unspaggettied" arhitecture

Reply:
Moving the functionality of helpers directory to Thunder is perhaps better off taken up as a separate activity, as it would involve changes to almost all the other rdkservices plugins.
@bobseamon, your thoughts on this?